generic/tests/cfg/stress_boot.cfg

```diff
@@ -3,11 +3,14 @@
     type = stress_boot
     max_vms = 5
     alive_test_cmd = uname -a
-    login_timeout = 240
+    login_timeout = 420
```

Comment:
@yiqianwei 
Hello, why we set the value as "420"? do it the limitation value for the guest boot up?
Do you try all windows guest, all guest can be boot up successfully?

Thanks
Jing

Reply:
> @yiqianwei Hello, why we set the value as "420"? 
For rhel9.5.0, Need to long time on different host

do it the limitation value for the guest boot up? 
I think it's a time period,  set login vm with timeout time

Do you try all windows guest, all guest can be boot up successfully?
Yes, Run win11 guest, PASS
 (2/2) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.stress_boot.q35: PASS (523.07 s)

Reply:
OK. understood